_pages/events/2025-03-27_project-end.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: single
+classes: wide
+title: HPC Ready - Developing RSE and HPC skills
+ 
+permalink: /events/2025-03-27_project-end/
+author_profile: false
+
+
+---
+
+**Join us for "HPC Ready: Developing RSE and HPC skills" on Thursday 27th March 2025 from 10:00 to 16:00 UTC**, which will take place **in person at the University of Edinburgh [Bayes Centre](https://bayes-centre.ed.ac.uk/), and online**.
+
+Everyone is welcome!
+
+### [Register now (form to be updated)](https://forms.office.com/Pages/ResponsePage.aspx?id=B3WJK4zudUWDC0-CZ8PTB5xW6C9sTFJAj9JEr0thFURUOTlJQ1RDUDlZUkxNTDJBMVdBSzRDOUlDQi4u)
+
+## Overview
+
+- 🕒 **Thursday 27th March** 2025, **[10:00 - 16:00 UTC](https://www.timeanddate.com/worldclock/fixedtime.html?msg=HPC+Ready%3A+Developing+RSE+and+HPC+skills&iso=20250327T10&p1=1440&ah=6)**
+- 🌍 In person at the University of Edinburgh, Bayes Centre and Online via Zoom
+
+UNIVERSE-HPC is organising an event to mark the end of the three years of the UNIVERSE-HPC project. We'll have talks from members of the project as well as the wider RSE and HPC training community. This will be a great opportunity to meet and network with members of the community and find out more about the work going on in this area! :sparkles:
```

Comment:
```suggestion
After three years, the UNIVERSE-HPC project is coming to a close and we are want to celebrate this milestone with an event about training for RSEs, with an emphasis on HPC. Members of the wider RSE and HPC training communities will give talks on their work in this field and UNIVERSE-HPC members will share some of our achievements over the course of the project. This will be a great opportunity to meet and network with members of the community and find out more about the work going on in the area of RSE training! ✨ 
```

Reply:
I don't think I can edit the suggested change above so just noting that in the edited version, the first line has a typo where it says "...we are want to....". You could perhaps say: "_After three years, the UNIVERSE-HPC project is coming to a close and we will be celebrating this milestone with an event..._".

Also, maybe expand HPC where it says with an emphasis on, i.e.: "_with an emphasis on High Performance Computing (HPC)._"?